### DISCUSSION TITLE
Audio files being rejected 'failed to process audio'

### DISCUSSION THREAD

**Discussion by counselinginsight** [deepgram/community#412] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

We've been using Deepgram for a couple of months now and we've recently been receiving a lot of errors and we want to get to the bottom of it. It's affecting our customers and causing frustration.

We're sending through seemingly working audio files that occasionally drop and give us a 'failed to process audio' error.

![image](https://github.com/deepgram/community/assets/143121278/580be423-a6c3-4392-b207-f4be47daa7b0)

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

/v1/listen?model=nova&filler_words=true&diarize=true&smart_format=true&detect_entities=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

f6ac8a92-5593-4042-a5bc-54ce524b71cb

### If possible, please attach your code or paste it into the text box.

```
          const readStream = fs.createReadStream(tempFilePath)

          const readStreamReady = new Promise((resolve, reject) => {
            readStream.on('ready', () => {
              resolve(true)
            })
            readStream.on('error', (err) => {
              reject(err)
            })
          })

          await readStreamReady

          // Send file to deepgram to transcribe
          const deepgram = new Deepgram(DEEPGRAM_API_KEY)
          const dgResponse = await deepgram.transcription.preRecorded(
            {
              stream: readStream,
              mimetype: mimetype,
            },
            {
              model: 'nova',
              filler_words: transcribe.type !== 'dictation',
              diarize: transcribe.type !== 'dictation',
              smart_format: true,
              // redact: true,
              detect_entities: transcribe.type !== 'dictation',
            }
          )
```

### If possible, please attach an example audio file to reproduce the issue.

[6543dd385e7ddd790d377fb1.webm](https://github.com/deepgram/community/assets/143121278/80e58e28-4795-4d07-934d-22dd0cfe4474)

**Comment by jpvajda** [comment]
@counselinginsight  Sorry this is a problem for you, what occurs if you try using the same audio without including the `detect_entities` parameter?

**Reply by counselinginsight** [reply]
I'll have to run it a few times on the server to try and get the errors. I'll get back to you with this. Thanks.

**Reply by counselinginsight** [reply]
Hey we need to use the detect entities for our platform. Why do you think detect entities would be causing an issue?

**Reply by jpvajda** [reply, marked as answer]
@counselinginsight Sorry for the very late reply. 

Detect Entities will invoke another model to process that request when the parameter is detected in the request. In my experience it sometimes can cause requests to fail.  Since it's been so long, you might want to try again as since this post we have done some improvements to how we handle entity detection.
